tests/test_mock_controller.py

```diff
@@ -620,7 +621,7 @@ async def test_move(self):
                     desired_value = end_position
                 else:
                     desired_value = 0
-                self.assertAlmostEqual(axis_telem[name], desired_value, msg=name)
+                assert axis_telem[name] == pytest.approx(desired_value)
 
 
 if __name__ == "__main__":
```

Comment:
I noticed you removed these lines from your other tests. Perhaps you could remove them from this one as well? I'll leave that for you to decide.